src/providers/ad/ad_gpo_child.c

```diff
@@ -778,6 +778,13 @@ main(int argc, const char *argv[])
         goto fail;
     }
 
+    if (sysvol_gpt_version < 0) {
+        DEBUG(SSSDBG_CRIT_FAILURE,
+              "get sysvol_gpt_version failed.[%d].\n",
```

Comment:
White space is missing before [%d] (if we need to print this value here at all).

Reply:
Hi @alexey-tikhonov I have added White space before [%d].